Android/shreyshrivastava/Intermediate/ToDo/app/src/main/java/com/example/todo/TaskAdapter.java

```diff
@@ -0,0 +1,114 @@
+package com.example.todo;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.CheckBox;
+import android.widget.CompoundButton;
+import android.widget.ImageView;
+import android.widget.Toast;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.google.android.gms.tasks.OnFailureListener;
+import com.google.android.gms.tasks.OnSuccessListener;
+import com.google.firebase.firestore.FirebaseFirestore;
+
+import java.util.List;
+
+public class TaskAdapter extends RecyclerView.Adapter<TaskAdapter.TaskViewHolder> {
+
+    Context context;
+    List<TaskModel> taskModelList;
+    FirebaseFirestore db;
```

Comment:
Update task functionality is missed. Please handle. Suggest you to add an update button in the list item view and when user clicks on it - load the item in edit text and update the entry in DB after user clicks on ADD button to save the changes

Reply:
[Update functionality.](https://github.com/applibgroup/assessment/pull/19/commits/1bdbd7ca6b5f2b5d62263dd7653dbaa6d483457d)